src/Backend/EdgeTPU/EdgeTPUToolchain.ts

```diff
@@ -127,12 +130,17 @@ class EdgeTPUCompiler implements Compiler {
 
     major = Number(major);
     minor = Number(minor);
-
-    if (isNaN(major) || isNaN(minor)) {
+    patch = Number(patch);
+    
+    if (isNaN(major) || isNaN(minor) || isNaN(patch)) {
       throw Error("Invalid version format.");
     }
 
-    return new Version(major, minor);
+    if (splitedVersion.length === 2 && !option) {
+      return new Version(major, minor);
+    }
+
+    return new Version(major, minor, patch, option);
```

Comment:
It seems like when the option is given and the patch is not given, this code returns the Version with 0 patch number.

Do I understand collectly?

Reply:
Yes. If version is `1.0-beta`, `parseVersion("1.0-beta")` returns `Version(1, 0, 0, "-beta")`.

Reply:
I see. Thanks.